mlir/lib/Dialect/ArmSVE/IR/CMakeLists.txt

```diff
@@ -10,5 +10,6 @@ add_mlir_dialect_library(MLIRArmSVEDialect
   LINK_LIBS PUBLIC
   MLIRIR
   MLIRLLVMDialect
+  MLIRVectorDialect
```

Comment:
What creates the dependency on Vector?

Reply:
Creating vector ops (e.g. `rewriter.create<vector::ExtractOp>(...)`)